lightning-invoice/src/lib.rs

```diff
@@ -796,17 +796,18 @@ impl SignedRawInvoice {
 ///
 /// The following example would extract the first B.
 /// ```
-/// use Enum::*;
+/// use lightning_invoice::{find_extract, find_all_extract};
```

Comment:
nit: only need to import one of them here and below.

Reply:
not technically true. Both need to be imported here, but find_all_extract is indeed sufficient below.